software/aliasvault.yml

```
@@ -0,0 +1,15 @@
+name: "AliasVault"
+website_url: "https://www.aliasvault.net"
+source_code_url: "https://gitlab.com/lanedirt/AliasVault"
```

Github is the preferred source code location due to being able to check the stars, last commit etc. As this is also linked on https://www.aliasvault.net/ and the gitlab 404s for me, I think GitHub would be better here.
```suggestion
source_code_url: "https://github.com/lanedirt/AliasVault"
```